snorkel/labeling/model/wrapper.py

```diff
@@ -0,0 +1,226 @@
+        l2: float = 0.0,
+        optimizer: str = "sgd",
+        optimizer_config: Optional[OptimizerConfig] = None,
+        lr_scheduler: str = "constant",
+        lr_scheduler_config: Optional[LRSchedulerConfig] = None,
+        prec_init: float = 0.7,
+        seed: int = np.random.randint(1e6),
+        log_freq: int = 10,
+        mu_eps: Optional[float] = None,
+        class_balance: Optional[List[float]] = None,
+        **kwargs: Any,
+    ) -> None:
+
+        self.cardinality = cardinality
+        self.verbose = verbose
+        self.device = device
+        self.metric = metric
+        self.tie_break_policy = tie_break_policy
+        self.n_epochs = n_epochs
+        self.lr = lr
+        self.l2 = l2
+        self.optimizer = optimizer
+        self.optimizer_config = (
+            optimizer_config
+            if optimizer_config is not None
+            else OptimizerConfig()  # type: ignore
+        )
+        self.lr_scheduler = lr_scheduler
+        self.lr_scheduler_config = (
+            lr_scheduler_config
+            if lr_scheduler_config is not None
+            else LRSchedulerConfig()  # type: ignore
+        )
+        self.prec_init = prec_init
+        self.seed = seed
+        self.log_freq = log_freq
+        self.mu_eps = mu_eps
+        self.class_balance = class_balance
+
+        self.label_model = LabelModel(
+            cardinality=self.cardinality, verbose=self.verbose, device=self.device
+        )
+
+    def fit(self, L: np.ndarray, Y: Optional[np.ndarray] = None) -> "SklearnLabelModel":
+        """
+        Train label model.
+
+        Parameters
+        ----------
+        L
+             An [n,m] matrix with values in {-1,0,1,...,k-1}
+        Y
+             Placeholder, not used for training model.
+
+        Returns
+        -------
+        SklearnLabelModel
+        """
+        self.label_model.fit(
+            L_train=L,
+            class_balance=self.class_balance,
+            n_epochs=self.n_epochs,
+            lr=self.lr,
+            l2=self.l2,
+            optimizer=self.optimizer,
+            optimizer_config=self.optimizer_config,
+            lr_scheduler=self.lr_scheduler,
+            lr_scheduler_config=self.lr_scheduler_config,
+            prec_init=self.prec_init,
+            seed=self.seed,
+            log_freq=self.log_freq,
+            mu_eps=self.mu_eps,
+        )
+
+        return self
+
+    def score(self, L: np.ndarray, Y: np.ndarray) -> float:
+        """Calculate score using self.metric.
+
+        Parameters
+        ----------
+        L
+             An [n,m] matrix with values in {-1,0,1,...,k-1}
+        Y
+             Gold labels associated with data points in L
+
+        Returns
+        -------
+        float
+            Score for metric speficied in self.metric
+        """
+
+        results = self.label_model.score(L, Y, [self.metric], self.tie_break_policy)
+        return results[self.metric]
+
+    def create_param_search_data(
```

Comment:
don't think this should be an instance method...doesn't use `self`. maybe a class method? but probably better as a top-level function

Reply:
made it static method - didn't need cls. mainly to avoid another import statement and also to package the wrapper and this method together. can revisit if it's too weird